lib/fizzy/execute.cpp

```diff
@@ -551,6 +552,18 @@ inline T ftrunc(T value) noexcept
     return std::trunc(value);
 }
 
+template <typename T>
+T fnearest(T a) noexcept
```

Comment:
This doesn't seem to enforce the same NaN rules as #481, yet the spec seems to be the same regarding it?

Reply:
We have them tested. I was expecting this will fail and I will need `if (isnan())` case, but seems not required. Will check against the C spec though.

Reply:
Did you check?

Reply:
This is the same as in other related functions: for NaN input it returns unspecified NaN.